ghost/i18n/locales/pa/newsletter.json

```diff
@@ -0,0 +1,28 @@
+{
+    "{date}": "{date}",
+    "Become a paid member of {site} to get access to all premium content.": "{site} ਦੇ ਸਹਿਯੋਗੀ ਮੈਂਬਰ ਬਣ ਕੇ ਸਾਰੀ ਪ੍ਰੀਮੀਅਮ ਸਮੱਗਰੀ ਤੱਕ ਪਹੁੰਚ ਪ੍ਰਾਪਤ ਕਰੋ।",
+    "By {authors}": "{authors} ਦੁਆਰਾ",
+    "Comment": "ਟਿੱਪਣੀ",
+    "complimentary": "ਮੁਫ਼ਤ",
+    "Email": "ਈਮੇਲ",
+    "free": "ਮੁਫ਼ਤ",
+    "Keep reading": "ਪੜ੍ਹਦੇ ਰਹੋ",
+    "Less like this": "ਇਸ ਵਰਗੇ ਘੱਟ",
+    "Manage subscription": "ਸਦੱਸਤਾ ਦਾ ਪ੍ਰਬੰਧਨ ਕਰੋ",
+    "Member since": "ਮੈਂਬਰ ਜਦੋਂ ਤੋਂ",
```

Comment:
Context check.  This appears in the footer of the email, as:  "Member since":  {date}